optima/makespreadsheet.py

```diff
@@ -681,9 +681,9 @@ def generate_const(self):
             'Unsuppressive ART',
             'Suppressive ART',
             'Probability of viral suppression on ART'],
-            [0.95, 0.58, 0.54, 0., 2.65, 0.9, 0.73, 0.5, 0.92, 0.9], 
-            [0.8, 0.47, 0.33, 0., 1.35, 0.82, 0.65, 0.3, 0.8, 0.8 ],
-            [0.98, 0.67, 0.68, 0.68, 5.19, 0.93, 0.8, 0.8, 0.95, 0.95],
+            [0.95, 0.58, 0., 2.65, 0.54, 0.9, 0.73, 0.5, 0.92, 0.9], 
```

Comment:
will have to check but upper limit for diagnosis behavior change of 5.19 looks wrong to me, if i'm reading that correctly